source/beamline/CLS/CLSJJSlit.cpp

```diff
@@ -62,37 +62,69 @@ void CLSJJSlitBladesControl::onStatusValueChanged()
 }
 
 // ======================== implementation for CLSJJSlit ======================================
+
+#include "beamline/CLS/CLSMAXvMotor.h"
+#include "beamline/CLS/CLSJJSlitGapControl.h"
+#include "beamline/CLS/CLSJJSlitCenterControl.h"
+
 CLSJJSlit::~CLSJJSlit()
 {
 
 }
 
-CLSJJSlit::CLSJJSlit(const QString &name, const QString &description, const QString &verticalBladesPVBaseName, const QString &horizontalBladesPVBaseName, double tolerance, double moveStartTimeoutSeconds, double limit, QObject *parent):
+CLSJJSlit::CLSJJSlit(const QString &name, const QString &description, const QString &upperBladePVName, const QString &lowerBladePVName, const QString &inboardBladePVName, const QString &outboardBladePVName, double tolerance, double moveStartTimeoutSeconds, double limit, QObject *parent) :
 	QObject(parent)
 {
 	name_ = name;
 	description_ = description;
 	limit_ = limit;
 
-	verticalBladesControl_ = new CLSJJSlitBladesControl(name+" Vertical", description+" Vertical", CLSJJSlitBladesControl::Vertical, verticalBladesPVBaseName, tolerance, moveStartTimeoutSeconds, this);
-	horizontalBladesControl_ = new CLSJJSlitBladesControl(name+" Horizontal", description+" Horizontal", CLSJJSlitBladesControl::Horizontal, horizontalBladesPVBaseName, tolerance, moveStartTimeoutSeconds, this);
-
-	connect(verticalBladesControl_, SIGNAL(connected(bool)), this, SLOT(onBladesControlConnected(bool)));
-	connect(verticalBladesControl_, SIGNAL(gapValueChanged(double)), this, SIGNAL(verticalGapValueChanged(double)));
-	connect(verticalBladesControl_, SIGNAL(centerValueChanged(double)), this, SIGNAL(verticalCenterValueChanged(double)));
-	connect(verticalBladesControl_, SIGNAL(statusStringChanged(QString)), this, SIGNAL(verticalStatusStringChanged(QString)));
-	connect(horizontalBladesControl_, SIGNAL(connected(bool)), this, SLOT(onBladesControlConnected(bool)));
-	connect(horizontalBladesControl_, SIGNAL(gapValueChanged(double)), this, SIGNAL(horizontalGapValueChanged(double)));
-	connect(horizontalBladesControl_, SIGNAL(centerValueChanged(double)), this, SIGNAL(horizontalCenterValueChanged(double)));
-	connect(horizontalBladesControl_, SIGNAL(statusStringChanged(QString)), this, SIGNAL(horizontalStatusStringChanged(QString)));
+	upperBlade_ = new CLSMAXvMotor(name+"UpperBlade", upperBladePVName, description+" UpperBlade", true, tolerance, moveStartTimeoutSeconds, this);
+	lowerBlade_ = new CLSMAXvMotor(name+"LowerBlade", lowerBladePVName, description+" LowerBlade", true, tolerance, moveStartTimeoutSeconds, this);
+	inboardBlade_ = new CLSMAXvMotor(name+"InboardBlade", inboardBladePVName, description+" InboardBlade", true, tolerance, moveStartTimeoutSeconds, this);
+	outboardBlade_ = new CLSMAXvMotor(name+"OutboardBlade", outboardBladePVName, description+" OutboardBlade", true, tolerance, moveStartTimeoutSeconds, this);
+
```

Comment:
Should there be a space before the literals in the first argument of the above 4 calls? 
ie.
```c++
upperBlade_ = new CLSMaxvMotor(name + " UpperBlade", ...);
```